src/home_robot/home_robot/motion/stretch.py

```diff
@@ -730,6 +735,18 @@ def validate(self, q=None, ignored=[], distance=0.0, verbose=False):
                 return False
         return True
 
+    def ros_pose_to_pinocchio(self, joint_angles):
+        pin_compatible_joints = np.zeros(9)
```

Comment:
this feels like something we could use a test for. just write a simple one which takes a hard coded Pin or ROS pose and converts it back and forth?

Reply:
Agreed. I will ad this.